libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -93,6 +104,18 @@ + (instancetype)sharedManager
         } else {
             sdkManager = [[self alloc] init];
         }
+
+        /*
+         * Checks if an analytics app name has already been set by the app.
+         * If not, fetches the default app name to be used and sets it.
+         */
+        NSString *currentAiltnAppName = [SalesforceSDKManager ailtnAppName];
```

Comment:
Not `init`, `initialize`.  It's the static equivalent of `init` functions, and will be called whatever the instance class is set to.